### ISSUE TITLE
after godot 3.3 it just doesn't show real ads

### ISSUE THREAD

**Issue by Roxy222** [Shin-NiL/Godot-Android-Admob-Plugin#163]
**Godot version: 3.3.2**

**AdMob Plugin version: 5.0.0**

**Issue description:**

**Comment by lfkdev** [comment]
Did you wait up to 24h? 
"Error code 3 (ERROR_CODE_NO_FILL) is a common issue with Admob, but out of the scope to this plugin. Here's the description on the API page: ERROR_CODE_NO_FILL: The ad request was successful, but no ad was returned due to lack of ad inventory."

**Comment by Roxy222** [comment]
Yes wait, I have a game made on godot 3.2 where the ads work perfectly, so to do a test I took the app id where the ads are normal and placed in the new game made on godot 3.3, but still continued error 3 , I don't know what I'm doing wrong because it's configured the same way as the other game

**Comment by surajhande** [comment]
This is happening to me right now. My project has been in production for almost a month and still don't see real ads.
It shows test ads on all phones.

**Comment by kamalasim** [comment]
how to set it to real

**Comment by bertangokalp** [comment]
How to set it to real please someone help me
